src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/HintExceptionHandler.kt

```diff
@@ -39,7 +39,7 @@ class HintExceptionHandler(private val errorCodeGenerator: ErrorCodeGenerator,
     @ExceptionHandler(NoHandlerFoundException::class)
     fun handleNoHandlerFoundException(error: Exception, request: WebRequest): Any
     {
-        logger.error(error.message)
+        logger.error(error.message, error)
```

Comment:
using the second argument to pass the error object means the stack trace gets logged